### DISCUSSION TITLE
Multiple certificates for same domain, how to deploy?

### DISCUSSION THREAD

**Discussion by Eddict** [acmesh-official/acme.sh#4984] [Q&A]
i have already an ECC certificate setup and running for my domain for a while, but i also needed an RSA version. so i created a new CSR, ran acme.sh with --signcsr parameter and all ok. acme.sh --list shows both certificates for same domain. one with KeyLength "4096" for the RSA one and one with "prime256v1" for the ECC one.
how can i deploy them separately? or is that just a matter of using the --ecc flag while deploying/(auto)renewing? i want them both to renew automatically but i am just not sure/afraid it will deploy the RSA one to my servers instead of the ECC one as they have the same domain (value). the RSA one should be deployed to only one server and not the rest. (using SSH to deploy)

**Comment by neilpang** [comment]
>> is that just a matter of using the --ecc flag while deploying/(auto)renewing? 

Yes.
